src/api/schema.graphql

```diff
@@ -2389,13 +2389,8 @@ type Query {
   """ get companies house information """
   getCompaniesHouseInformation(companiesHouseNumber: String!): CompaniesHouseResponse
 
-<<<<<<< HEAD
   """ get Ordnance Survey address """
   getOrdnanceSurveyAddress(postcode: String!, houseNameOrNumber: String!): OrdnanceSurveyResponse
-=======
```

Comment:
this might cause a merge conflict as was fixed in your previous PR?

Reply:
I'm not following, this is resolving a merge conflict?